hector_quadrotor_controller_gazebo/src/quadrotor_hardware_gazebo.cpp

```diff
@@ -144,12 +147,13 @@ void QuadrotorHardwareSim::readSim(ros::Time time, ros::Duration period)
   callback_queue_.callAvailable();
 
   // read state from Gazebo
-  const double acceleration_time_constant = 0.1;
   gz_pose_             =  link_->GetWorldPose();
-  gz_acceleration_     = ((link_->GetWorldLinearVel() - gz_velocity_) + acceleration_time_constant * gz_acceleration_) / (period.toSec() + acceleration_time_constant);
+  gz_acceleration_     = link_->GetWorldLinearAccel();
```

Comment:
Please hold on this, need to change back to using your differential update method, https://bitbucket.org/osrf/gazebo/issue/243/getlinearacceleration-always-returns-zeros